pkg/frontend/querymiddleware/codec.go

```diff
@@ -531,10 +536,10 @@ func DecodeRangeQueryTimeParams(reqValues *url.Values) (start, end, step int64,
 }
 
 func instantTimeParamNow() int64 {
-	return time.Now().UnixMilli()
+	return time.Now().UTC().UnixMilli()
```

Comment:
is the `UTC()` necessary? I thought `UnixMilli` is only defined in UTC

Reply:
Ah yeah true. I am just in the habit of making sure anything with time gets UTC-ed